test/controllers/sessions_controller_test.rb

```diff
@@ -1,7 +1,24 @@
 require 'test_helper'
 
 class SessionsControllerTest < ActionController::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  include SessionsHelper
+  
+  def setup
+    @user = User.new(first_name:"apoorva",last_name:"lal",
+                      email:"foo@bar.com",password:"123456",
+                      password_confirmation:"123456")
+  end
+  
+  test "valid users able to log in" do
+    @user.save
+    log_in @user
+    assert logged_in?
+    assert current_user == @user
+  end
+  
+  test "can't log in to a non-existing user" do
+    log_in @user
+    assert_not logged_in?, "User #{@user.first_name} #{@user.last_name} doesn't exist."
```

Comment:
So the message doesn't seem necessary. It's nice. But it's encapsulated in the test name.

Remember, tests are solely for development purposes. They're not used in a production environment or seen by a non-developer.